src/ext/dispatch_compat_php5.c

```diff
@@ -281,4 +285,87 @@ BOOL_T ddtrace_should_trace_call(zend_execute_data *execute_data, zend_function
 
     return TRUE;
 }
+
+/**
+ * trace.c
```

Comment:
Ah! I wanted to ask you about this. Essentially, as we start thinking of how we want to separate the code for each major PHP version, this would be related to the code in **trace.c**. So for example if we had folders this could be put in **php5/trace.c** and **php7/trace.c**.

We probably don't want to figure out how to separate the code for this PR, but perhaps you, me and @morrisonlevi could have a chat about it sometime soon? :)